File: 06/6.java
class main  
{
    public static void main(String[] args) 
    {
        int sum = 0 , square = 0;
        for (int i = 1 ; i <= 100 ; i++)
        {
            sum += i;
            square += i*i;
        }
        System.out.println("Answer: "+ ((sum*sum) - square));
    }
}

File: 06/6.c
#include <stdio.h>

int main()
{
	long long i , sum = 0 , square = 0;
	for (i = 1 ; i <= 100 ; i++)
	{
		sum += i;
		square += i * i;
	}
	printf("%lld\n" , (sum * sum) - square);
}
